website/public/js/controllers/menuCtrl.js

```diff
@@ -41,7 +45,7 @@ angular.module('habitrpg')
       };
 
       $scope.hasNoNotifications = function() {
-        return selectNotificationValue(false, false, false, false, true);
+        return selectNotificationValue(false, false, false, false, false, true);
```

Comment:
uuuuuuuuuuuuuuuuuuuuuuuugh

Not something to do now, but this should be refactored to take in a hash instead.